### PR TITLE
tests(std,Array): Add missing tests for Array#lastIndexOf

### PR DESCRIPTION
**Pull request by saulecabrera**
Adds missing test for `Array#lastIndexOf`. On that same note, I took the time to verify which methods are untested for Arrays, and it's only `lastIndexOf` and `slice`.

I'm working on the tests for `slice` separately. 

- [x] I've read the contributing guidelines

### CODE REVIEW

**Comment by MaxGraey** [comment]
nice! Thanks!

File: tests/compiler/std/array.ts [hunk 1]
```diff
@@ -313,6 +313,19 @@ var i: i32;
...
 }
 
+// Array#lastIndexOf
+
+{
+  const numbers: i32[] = [2, 5, 9, 2];
+  assert(numbers.lastIndexOf(2) == 3);
+  assert(numbers.lastIndexOf(7) == -1);
+  assert(numbers.lastIndexOf(2, 3) == 3);
+  assert(numbers.lastIndexOf(2, 2) == 0)
```
Comment (MaxGraey) [inline tests/compiler/std/array.ts:323]:
```suggestion
  assert(numbers.lastIndexOf(2, 2) == 0);
```

Comment (saulecabrera) [reply tests/compiler/std/array.ts]: Yeah good catch, this was causing CI to fail.

**Review by MaxGraey — APPROVED** [review]

**Comment by MaxGraey** [comment]
plz update fixtures for array: `npm run test:compiler std/array -- --create`

**Comment by saulecabrera** [comment]
> plz update fixtures for array: `npm run test:compiler std/array -- --create`

Yeah, forgot to re-run tests after rebasing master

**Comment by dcodeIO** [comment]
Great, thank you! :)

**Pull request merged**